src/util/system.h

```diff
@@ -94,6 +95,15 @@ fs::path GetDefaultDataDir();
 // The blocks directory is always net specific.
 const fs::path &GetBlocksDir();
 const fs::path &GetDataDir(bool fNetSpecific = true);
+
+/**
+ * Get data directory path.
+ *
+ * @param      argsManager Arguments manager instance.
+ * @param      fnetSpecific Append network identifier to the path.
+ * @param[out] data Directory path.
+ */
+void GetDataDir(ArgsManager& argsManager, bool fNetSpecific, fs::path* path);
```

Comment:
Note: Is signature in form: `void GetDataDir(fs::path* path, ArgsManager& argsManager, bool fNetSpecific = true);` preferred?